### DISCUSSION TITLE
Shutterstock watermark

### DISCUSSION THREAD

**Discussion by quintendewilde** [deforum-art/sd-webui-text2video#133] [Q&A] [answered]
I've run the colab: https://colab.research.google.com/github/camenduru/stable-diffusion-webui-colab/blob/main/video/stable/stable_diffusion_1_5_video_webui_colab.ipynb#scrollTo=SaAJk33ppFw1

Then chose the txt 2 video option
And ran the parameters with a prompt: Woman dancing for example

Running Colab Pro with premium

And getting watermarks? 
<img width="516" alt="Schermafbeelding 2023-04-22 om 15 07 25" src="https://user-images.githubusercontent.com/6031569/233787010-0d6104cb-a11f-4fe4-8c64-da0149dfd72b.png">

**Comment by kabachuha** [comment]
Processing the same video in the vid2vid mode helps to remove the watermark quite often

**Comment by aivrar** [comment]
Use this model instead:
https://huggingface.co/kabachuha/animov-0.1-modelscope-original-format

No more Shutterstock watermark, no need for extra processing.

**Reply by quintendewilde** [reply]
How do I add the model?

**Reply by aivrar** [reply]
You can just replace all those 4 files of the model with all the 4 files your using now. Save your old files somewhere of course if it doesn't work out, but it will.

**Reply by quintendewilde** [reply]
To be sure these models I replace?
<img width="291" alt="Schermafbeelding 2023-04-22 om 15 56 44" src="https://user-images.githubusercontent.com/6031569/233789240-67d1ba8a-602f-49f7-9efe-fece61b14f6e.png">

Side question, when my videos are generating black images? 
Is this the NSFW filter that thinks my prompts are not allowed?

**Reply by aivrar** [reply]
yes those 4 files, note they are the same file names as the model I was showing you.

**Comment by aivrar** [comment]
I should have given this link here too 
https://github.com/deforum-art/sd-webui-text2video#fine-tunes-and-how-to-use-them

**Reply by quintendewilde** [reply]
Check, thanks for this!

**Reply by quintendewilde** [reply]
Crap, I'm new at this. 

But none of the files in this repo are similar to the one in my colab.
<img width="1594" alt="Schermafbeelding 2023-04-22 om 16 04 35" src="https://user-images.githubusercontent.com/6031569/233789575-970d4927-9109-4a4e-b3fa-1bc1eecf3d64.png">

**Reply by aivrar** [reply]
oh oh, you're using colab huh? yeah I know this problem using the gui with Docker. Since many unpdates I'm not sure where, you have may have to to do some manual pathing hacks. I'm not sure at the moment what since I stopped using Docker.

**Reply by quintendewilde** [reply]
Ok I'll take a look at it. Thanks for the help though.

**Reply by aivrar** [reply, marked as answer]
Hey I was thinking, since you got pro colab you may as well so the vid2vid method as mentioned by kabachuha above. you got that sweet processing power the sky's the limit.

**Comment by camenduru** [comment]
Hi @quintendewilde please try this https://colab.research.google.com/github/camenduru/text-to-video-synthesis-colab/blob/main/watermark_remover_gradio.ipynb

test here
https://discord.com/channels/1011314943380959313/1094146924044628048/1094203218021580831
